src/components/artisttimeline/ArtistTimeline.jsx

```diff
@@ -1,38 +1,23 @@
 import React from 'react';
-// import PropTypes from 'prop-types';
+import PropTypes from 'prop-types';
 import { Timeline, TimelineItem } from 'vertical-timeline-component-for-react';
 
-const inputdata = [
-  {
-    date: '1982',
-    description: 'Est incididunt sint eu minim dolore mollit velit velit commodo ex nulla',
-  },
-  {
-    date: '1985',
-    description: 'Est incididunt sint eu minim dolore mollit velit velit commodo ex nulla',
-  },
-  {
-    date: '1992',
-    description: 'Est incididunt sint eu minim dolore mollit velit velit commodo ex nulla',
-  },
-  {
-    date: '2000',
-    description: 'Est incididunt sint eu minim dolore mollit velit velit commodo ex nulla',
-  },
-];
-
-const ArtistTimeline = () => (
+const ArtistTimeline = ({ inputData }) => (
   <Timeline lineColor="#ddd">
-    {inputdata.map(item => (
+    {inputData.map(item => (
       <TimelineItem
-        key={item.date}
-        dateText={item.date}
+        key={item.node.date}
+        dateText={item.node.date}
         style={{ color: '#e86971' }}
       >
-        <p>{item.description}</p>
+        <p>{item.node.description.description}</p>
```

Comment:
description.description - rewrite first description: *.description -> info.description -> artist.description -> buildings.description  and etc.

Reply:
It does not work like that, there is a field "description" which handles the text, and it can be queried only in this way. We will use destructuring to avoid repetition when refactoring components.